src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -143,7 +143,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'required' => true,
         ])
             ->add('inputPackItems', TypeaheadProductPackCollectionType::class, [
-                'remote_url' => $this->context->getLegacyAdminLink('AdminProducts', true, [], ['ajax' => 1, 'action' => 'productsList', 'forceJson' => 1, 'excludeVirtuals' => 1, 'limit' => 20]) . '&q=%QUERY',
+                'remote_url' => $this->context->getLegacyAdminLink('AdminProducts', true, ['ajax' => 1, 'action' => 'productsList', 'forceJson' => 1, 'excludeVirtuals' => 1, 'limit' => 20]) . '&q=%QUERY',
```

Comment:
Do you know why `'&q=%QUERY'` is not in query parameters?

Reply:
Because it would be sanitized, preventing the existing js to replace the placeholder with an actual parameter